scripts/release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../"
+cd ${ROOT_DIR}
+
+print_usage() {
+  cat <<EOF
+Usage: $(basename $0) tag"
+
+Tag example: v1.2.3
+EOF
+}
+
+if [[ $# != 1 ]]
+then
+  print_usage
+  exit 1
+fi
+
+release_tag="$1"
+
+import_gpg_keys() {
+  echo ">>> Setting GnuPG configuration ..."
+  mkdir -p ~/.gnupg
+  chmod 700 ~/.gnupg
+  cat > ~/.gnupg/gpg.conf <<EOF
+no-tty
+pinentry-mode loopback
+EOF
+
+  echo ">>> Importing secret key ..."
+  echo "$GPG_SECRET_KEY" > seckey.gpg
+  trap "rm seckey.gpg" EXIT
```

Comment:
This will not remove file if script gets killed. Use more signals, at least `KILL`

Reply:
Good point, will add 👍